src/Draco.Compiler/Internal/Symbols/Metadata/MetadataSymbol.cs

```diff
@@ -108,4 +109,63 @@ internal static class MetadataSymbol
     private static FunctionSymbol SynthetizeConstructor(
         MetadataTypeSymbol type,
         MethodDefinition ctorMethod) => new SynthetizedMetadataConstructorSymbol(type, ctorMethod);
+
+    public static string GetDocumentation(MetadataAssemblySymbol assembly, string documentationName)
+    {
+        var root = assembly.AssemblyDocumentation?.DocumentElement;
+        var xml = root?.SelectSingleNode($"//member[@name='{documentationName}']")?.InnerXml ?? string.Empty;
+        return string.Join(Environment.NewLine, xml.ReplaceLineEndings("\n").Split('\n').Select(x => x.TrimStart()));
+    }
```

Comment:
Since you take the assembly and the doc name for the same metadata symbol, why not take the symbol directly and query it for these 2 instead of explicitly asking for them?